doc/guides/writing-tests.md

```diff
@@ -231,12 +231,14 @@ assert.throws(
 
 For performance considerations, we only use a selected subset of ES.Next
 features in JavaScript code in the `lib` directory. However, when writing
-tests, it is encouraged to use ES.Next features that have already landed
-in the ECMAScript specification. For example:
+tests, for the ease of backporting, it is encouraged to use those ES.Next
+features that can be used directly [without a flag in v4.x](http://node.green).
```

Comment:
May be `s/v4.x/all maintained branches` for timelessness. (we can add a link to https://github.com/nodejs/lts to explain what are those maintained branches)